### DISCUSSION TITLE
Wait for Github actions to finish

### DISCUSSION THREAD

**Discussion by sqveeze** [vercel/vercel#8619] [Help] [answered]
Hello,

I would like to ask if there's a way to hold the vercel deployment until the github actions finishes? It would be nice to let GitHub actions run the linter, tests etc and only after this process(if successful), vercel deploy the site.

Tried to hack with ignored build steps and also tried to disable github in config and then run the deploy webhook but none of these worked so far.

**Comment by amyegan** [comment, marked as answer]
Hi @sqveeze! If you're using GitHub Actions for something else already, you could move the whole deployment to that workflow. See "[How can I use GitHub Actions with Vercel?](https://vercel.com/guides/how-can-i-use-github-actions-with-vercel)" for a walkthrough.

Setting [github.enabled](https://vercel.com/docs/project-configuration#git-configuration/github-enabled) to `false` in `vercel.json` should prevent [Vercel for GitHub](https://vercel.com/docs/concepts/git/vercel-for-github) from deploying the project. If that is not working for you, we may have a bug to investigate 😨 🐛 Can you share your config? I want to see if I can replicate the problem.

**Reply by sqveeze** [reply]
Thank you, that guide works flawless. The github.enabled false is working itself, i meant to say that if you set it to false and call the deploy hook, that will not work.

**Reply by irangarcia** [reply]
Hey @amyegan. Can I set `github.deploymentEnabled` to `false` and still have the bot commenting on the PR's? We have all the deployment on the Github Actions workflow

**Reply by amyegan** [reply]
Sorry for the late response. You've probably already tried it yourself at this point, but setting `deploymentEnabled` to false will also stop the automatic comments on PRs for disabled branches.
